cookbook/console/style.rst

```diff
@@ -0,0 +1,371 @@
+.. index::
+   single: Console; Style commands
+
+How to Style a Console Command
+==============================
+
+One of the most boring tasks when creating console commands is to deal with the
+styling of the command's input and output. Displaying titles and tables or asking
+questions to the user involves a lot of repetitive code.
+
+Consider for example the code used to display the title of the following command::
+
+    // src/AppBundle/Command/GreetCommand.php
+    namespace AppBundle\Command;
+
+    use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+    use Symfony\Component\Console\Input\InputInterface;
+    use Symfony\Component\Console\Output\OutputInterface;
+
+    class GreetCommand extends ContainerAwareCommand
+    {
+        // ...
+
+        protected function execute(InputInterface $input, OutputInterface $output)
+        {
+            $output->writeln('<info>Lorem Ipsum Dolor Sit Amet</>');
+            $output->writeln('<info>==========================</>');
+            $output->writeln('');
```

Comment:
I think it's not fair to use this, as it can also be done by passing an array to a single writeln method call.